blocks/abp-review/utils/sendHelixData.js

```diff
@@ -21,12 +23,18 @@ const sendHelixData = ({
   if (visitorId) {
     data.push({ name: 'VisitorId', value: visitorId });
   }
+
   if (page) {
     data.push({ name: 'Page', value: page });
   }
+
   if (timeSpentInSeconds) {
     data.push({ name: 'TimeSpentInSeconds', value: timeSpentInSeconds });
   }
+  const profileDetails = getLocalStorage('profile');
+  if (profileDetails) {
+    data.push({ name: profileDetails?.name, email: profileDetails?.email });
```

Comment:
`?` after `profileDetails` may be irrelevant since you are already checking for presence of `profileDetails` on L#35